modules/dmrpp_module/tests/builddmrpp/ghrsst_cmr_dmrpp.dap

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<request xmlns:bes="http://xml.opendap.org/ns/bes/1.0#" reqID="[thread:http-nio-8080-exec-4-42][bes_client:/-2]">
+    <setContext name="dap_format">dap4</setContext>
+    <setContext name="invocation">http://localhost:8080/opendap/hyrax/build_dmrpp/collections/C2036877806-POCLOUD/granules/20220812010000-OSISAF-L3C_GHRSST-SSTsubskin-GOES16-ssteqc_goes16_20220812_010000-v02.0-fv01.0</setContext>
+    <!-- setContainer name="c" space="catalog">/data/grid_1_2d.h5.dds</setContainer -->
+    <setContainer name="c" space="builddmrpp">/collections/C2036877806-POCLOUD/granules/20220812010000-OSISAF-L3C_GHRSST-SSTsubskin-GOES16-ssteqc_goes16_20220812_010000-v02.0-fv01.0</setContainer>
+    <define name="d">
+	   <container name="c" />
+    </define>
+    <get type="dap" definition="d" returnAs="dmrpp"/>
```

Comment:
Can we use '<get type="dmr" ... returnAs="dmrpp"/>' instead of 'type="dap"'?

Asking because there is probably something I don't understand about the behavior of 'returnAs.'

Reply:
Also have the same question as  James.  Why not using "get type="dmr",

Reply:
Most likely, haven't tested that but will.  Used pre-existing BESCMD example as the basis for the test.

Reply:
Actually, it appears not, at least as currently implemented:

<Message>Unable to transmit response, no transmitter for dmr</Message>

What is the most appropriate request type that the Transmitter (.i.e., returnAs="dmrpp") should be coupled to?